polynote-frontend/polynote/cell.js

```diff
@@ -350,17 +362,36 @@ export class CodeCell extends Cell {
         const rel = args.rel || 'none';
         const lang = args.lang || null;
         const self = this;
-        CodeCell.parseContent(content, mimeType, lang).then(function(result) {
-            self.cellOutputDisplay.appendChild(
-                div(['output'], result).attr('rel', rel).attr('mime-type', mimeType)
-            );
+        return CodeCell.parseContent(content, mimeType, lang).then(function(result) {
+            const el = div(['output'], result).attr('rel', rel).attr('mime-type', mimeType);
+            self.cellOutputDisplay.appendChild(el);
+            return el;
         }).catch(function(err) {
             self.cellOutputDisplay.appendChild(
                 div(['output'], err)
             );
         });
     }
 
+    addResult(result) {
+        if (result instanceof ResultValue) {
+            // TODO: keep "result" and "output" separate for UI... have a way to show declarations, results, outputs, etc. separately
+            if (result.name !== "Out") {
```

Comment:
If a user declares a `val Out = ...` will it show up here?

If so, maybe that's not necessarily a bad thing - maybe I want to hook into the automatic repr display for `Out`s for a value that for some reason can't be the return value of the cell...?

Reply:
Yeah, nothing stopping you from declaring `val Out`... but then if you also have an expression as the last line, it would be a compile error for redeclaring that. Not super worried about it right now, but maybe that's a case we should detect (and then don't capture the expression, sorry 'bout it)